plugin/auth_ldap/src/auth_ldap_impl.cc

```diff
@@ -188,7 +224,7 @@ groups_t AuthLDAPImpl::search_ldap_groups(const std::string &user_dn) {
   log_srv_dbg("AuthLDAPImpl::search_ldap_groups");
   groups_t list;
 
-  std::shared_ptr<Connection> conn = pool_->borrow_connection();
+  std::shared_ptr<Connection> conn = pool_->borrow_connection(true);
```

Comment:
default_connect already defaults to true, any reason to enforcing it now?

Reply:
No, this was a debugginging leftover.